webpack/assets/javascripts/react_app/components/HostDetails/index.js

```diff
@@ -53,7 +53,6 @@ const HostDetails = ({
     `/api/hosts/${id}`,
     HOST_DETAILS_API_OPTIONS
   );
-
   const isNavCollapsed = useSelector(selectIsCollapsed);
```

Comment:
I'm bit confused, you've removed it previously and it worked, why is it now needed again?

Reply:
I added it back to satisfy when the tabs component needs the horizontal scroll functionality.
I need to know if this is true or false for the correct pixel change to be applied on said component.

Reply:
oh, right :) get it :+1: and I liked that we would get rid of it :( xD